spec/features/contestants/index_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe "Contestant Index Page" do
+  before(:each) do
+    @recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+    @furniture_challenge = Challenge.create(theme: "Apartment Furnishings", project_budget: 1000)
+    
+    @news_chic = @recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+    @boardfit = @recycled_material_challenge.projects.create(name: "Boardfit", material: "Cardboard Boxes")
+    
+    @upholstery_tux = @furniture_challenge.projects.create(name: "Upholstery Tuxedo", material: "Couch")
+    @lit_fit = @furniture_challenge.projects.create(name: "Litfit", material: "Lamp")
+    
+    @jay = Contestant.create(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+    @gretchen = Contestant.create(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+    @kentaro = Contestant.create(name: "Kentaro Kameyama", age: 30, hometown: "Boston", years_of_experience: 8)
+    @erin = Contestant.create(name: "Erin Robertson", age: 44, hometown: "Denver", years_of_experience: 15)
+
+    ContestantProject.create(contestant_id: @jay.id, project_id: @news_chic.id)
+    ContestantProject.create(contestant_id: @gretchen.id, project_id: @news_chic.id)
+    ContestantProject.create(contestant_id: @gretchen.id, project_id: @upholstery_tux.id)
+    ContestantProject.create(contestant_id: @kentaro.id, project_id: @upholstery_tux.id)
+    ContestantProject.create(contestant_id: @kentaro.id, project_id: @boardfit.id)
+    ContestantProject.create(contestant_id: @erin.id, project_id: @boardfit.id)
+  end
+
+  it 'when visiting the contestant index page, user can see all contestant names' do
+    visit "/contestants"
+    expect(page).to have_content(@jay.projects.first.name)
+    expect(page).to have_content(@gretchen.projects.first.name)
+    expect(page).to have_content(@kentaro.projects.first.name)
+    expect(page).to have_content(@erin.projects.first.name)
+  end
```

Comment:
Try to use `within` blocks to help test where something should be on a page. 
```ruby
within "#contestants_list" do
 expect(page).to have_content(@jay.name)
 expect(page).to have_content(@gretchen.name)
 expect(page).to have_content(@kentaro.name)
 expect(page).to have_content(@erin.name)
end
```
Then in your view you would just need to have css id called `contestants_list`. 

```ruby
<ul id="contestants_list"> <--- here is where it will look
  <% @contestants.each do |contestant| %>
  <li id="contestant_<%= contestant.id %>"><%= contestant.name %> <br/>
    Projects: <%= contestant.projects_list %>
  </li>
  <% end %>
</ul>
```